packages/strapi-design-system/src/Checkbox/Checkbox.tsx

```diff
@@ -24,10 +23,18 @@ const CheckboxTick = (props) => {
   return <BaseCheckbox id={id} {...props} />;
 };
 
-export const Checkbox = ({ children, disabled, id, hint, error, ...props }) => {
+interface CheckboxProps {
```

Comment:
This should `extend BaseCheckboxProps` because we spread the rest of props to it.

Reply:
ok done thanks